tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -61,6 +64,10 @@ describe(__filename, () => {
     });
   });
 
+  afterEach(() => {
+    clock.restore();
```

Comment:
If we're only using the clock in one test, is there a reason we have to do this in `afterEach`?

Reply:
sure

Reply:
altho - i don't think it would hurt having this here... I am just thinking if another tests is added with a clock..

I also see clock some other files where it appears only to be used 1 time and has this similar set up...